examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -9,13 +9,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# SASRec\n",
+    "# SASRec & SSEPT\n",
     "\n",
-    "### Self-Attentive Sequential Recommendation Using Transformer \\[1\\]\n",
+    "### Sequential Recommendation Using Transformer \\[1, 6\\] \n",
     "\n",
     "![image.png](attachment:image.png)\n",
     "\n",
-    "This is a class of sequential recommendation that uses Transformer \\[2\\] for encoding the users preference represented in terms of a sequence of items purchased/viewed before. Instead of using CNN (Caser \\[3\\]) or RNN (GRU4Rec \\[4\\], SLI-Rec \\[5\\] etc.) the approach relies on Transformer based encoder that generates a new representation of the item sequence. This notebook provides an example of necessary steps to train and test a SASRec model. "
+    "This is a class of sequential recommendation that uses Transformer \\[2\\] for encoding the users preference represented in terms of a sequence of items purchased/viewed before. Instead of using CNN (Caser \\[3\\]) or RNN (GRU4Rec \\[4\\], SLI-Rec \\[5\\] etc.) the approach relies on Transformer based encoder that generates a new representation of the item sequence. There are two variants of this Transformer based approach included here, (a) based on vanilla Transformer that models only the item sequence, called SASRec \\[1\\] and (b) personalized Transformer, called SSE-PT, \\[6\\]) that also models the users. This notebook provides an example of necessary steps to train and test a SASRec model. "
```

Comment:
`This notebook provides an example of necessary steps to train and test a SASRec model.`
here we are also using SSE-PT, we could say:
`This notebook provides an example of necessary steps to train and test a SASRec model or a SSE-PT model.`

Reply:
Modified.